src/xcode/ENA/ENA/Source/Models/Task Scheduling/ENATaskScheduler.swift

```diff
@@ -59,20 +59,26 @@ final class ENATaskScheduler {
 		let identifierString = taskIdentifier.backgroundTaskSchedulerIdentifier
 		BGTaskScheduler.shared.register(forTaskWithIdentifier: identifierString, using: .main) { task in
 			self.scheduleTask()
+			let backgroundTask = DispatchWorkItem {
+				execute(task)
+			}
```

Comment:
The reason you import DispatchWorkItem is because you want to cancel the task later or get the completion signal?

Reply:
I introduced the DispatchWorkItem here in order to allow future cancellation of the BG task and its subtasks. This is currently not really functional since the underlying tasks do not allow preemptive cancellation. However, for the exposure detection task interruption without cancellation may be a source or undefined behavior or bugs as described by chris k. I therefore wanted to prepare the background tasks to be able to forward the cancel signal that is provided by the expirationHandler.